examples/demos/interfaces_query_demo/main.go

```diff
@@ -40,7 +45,22 @@ func main() {
 	queryChart(client)
 }
 
-func createInterface(client *kentikapi.Client) models.ID {
+func createDevice(client *kentikapi.Client) models.ID {
+	device := models.NewDeviceDNS(
+		"testapi_for_demo",
```

Comment:
```suggestion
		"interfaces_query_demo_device",
```
To avoid conflicts and help with manual cleanup, if needed

Reply:
Done